src/apps/loan-list/materials/selectable-material/selectable-material.tsx

```diff
@@ -48,7 +48,24 @@ const SelectableMaterial: FC<SelectableMaterialProps & MaterialProps> = ({
   } = material || {};
 
   return (
-    <li className="arrow arrow__hover--right-small">
+    <li
+      className="arrow arrow__hover--right-small cursor-pointer"
+      // Eslint is right, this is not an interactive element. But the customer wants it to be.
+      // Therefore we assign it the role of a button and add the necessary event handlers.
+      // eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role
+      role="button"
```

Comment:
Appreciate the comment but consider dropping the `role="button"` attribute.

This may cause nested button elements which is not allowed. Consequently we have *not* assigned roles to [elements in the card list component](https://github.com/reload/dpl-react/blob/cbb03875ea3cc1aa93bb9030dd54a8cbc1764abd/src/components/card-item-list/card-list-item/card-list-item.tsx#L108-L127).

Reply:
👍